packages/paste-core/utilities/grid/__tests__/grid.test.tsx

```diff
@@ -0,0 +1,588 @@
+import * as React from 'react';
+import renderer from 'react-test-renderer';
+import {DefaultTheme} from '@twilio-paste/theme-tokens';
+import {Theme} from '@twilio-paste/theme';
+import {Space} from '@twilio-paste/style-props';
+import {Box} from '@twilio-paste/box';
+import {Grid, Column} from '../src';
+import {getOuterGutterPull, getStackedColumns, getColumnOffset, getColumnSpan} from '../src/helpers';
+
+describe('Grid Unit Tests', () => {
+  it('it should return: -0.25rem', (): void => {
+    const mockGutter: Space = 'space20';
+    expect(getOuterGutterPull(DefaultTheme, mockGutter)).toStrictEqual('-0.25rem');
```

Comment:
These tests seem.... fragile.  If we change the token, the test will break right?  I don't think returning `-0.25rem` matters to whether Grid works well or not, does it?  Can we test less precise things?  Essentially, any test that expects a token to be a specific value is fragile.

Reply:
Is it worth unit testing this function? Right it takes a token, and returns the negative value of that token. If anything other than a token else is passed to it, it returns undefined.

Reply:
We should test the functionality, but you should mock the inputs. You always want to make sure that give a gutter, it returns the negative value of that gutter as a key from the space object on a theme. And given an array of gutters, it returns an array of values that match the keys on the theme object.

What theme doesn't matter, what gutters doesn't matter.

Reply:
```ts
const mockTheme = {
  space: {
    space10: '2px',
    space20: '4px',
    space30: '6px',
    space40: '8px'
  }
}
const mockGutter = 'space20'
cosnt mockGutters = ['space10', 'space30', 'space20']

describe('Grid Unit Tests', () => {

  it('should return a single negative margin value', (): void => {
    expect(getOuterGutterPull(mockTheme, mockGutter)).toStrictEqual('-4px');
  }

  it('should return a responsive set of negative margin values', (): void => {
    expect(getOuterGutterPull(mockTheme, mockGutters)).toStrictEqual(['2px', '6px', '4px']);
  }

}
```

Reply:
This makes more sense now that I'm seeing the example. Thanks.